mailer.js

```diff
@@ -263,6 +255,52 @@ module.exports = function (log) {
     })
   }
 
+
+  Mailer.prototype.authorizeLoginEmail = function (message) {
+    log.trace({ op: 'mailer.authoirzeLoginEmail', email: message.email, uid: message.uid })
+
+    var templateName = 'authorizeLoginEmail'
+    var query = {
+      code: message.code,
```

Comment:
These work. I'm find with `code`. Or maybe it feels cleaner all being the same. I don't lean either way, other than the easier way, which may be "just leave it".